### PR TITLE
storage: use sessionStorage for the communication with cockpit-storage

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: src/components/storage/ModifyStorage.jsx [hunk 1]
```diff
@@ -41,12 +41,12 @@ export const ModifyStorage = ({ idPrefix, onCritFail, onRescan, setShowStorage,
               variant="link"
               icon={<WrenchIcon />}
               onClick={() => {
-                  window.localStorage.setItem("cockpit_anaconda",
-                                              JSON.stringify({
-                                                  mount_point_prefix: targetSystemRoot,
-                                                  available_devices: selectedDevices,
-                                                  efi: isEfi,
-                                              })
+                  window.sesionStorage.setItem("cockpit_anaconda",
```
Comment (martinpitt) [inline src/components/storage/ModifyStorage.jsx:44]: ses*s*ion I suppose

File: test/vm.install [hunk 2]
```diff
@@ -49,7 +49,9 @@ def vm_install(image, verbose, quick):
             # cockpit-storaged is also available in the default rawhide compose, make sure we don't pull it from there
             download_from_copr(f"packit/cockpit-project-cockpit-{cockpit_pr}", "cockpit-storaged", machine)
         else:
-            packages_to_download += " cockpit-storaged"
+            # FIXME: Download cockpi-storaged from custom COPR
+            # till https://github.com/cockpit-project/cockpit/pull/19924#event-11785480839 is released
+            download_from_copr("@cockpit/main-builds", "cockpit-storaged", machine)
```
Comment (martinpitt) [inline test/vm.install:54]: FTR, it landed in F38/39 but I held back 40/rawhide for this. So this is good for now.

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Comment by martinpitt** [comment]
The bot is now running `python3 -B test/common/run-tests --jobs 2`, so that worked.

**Comment by KKoukiou** [comment]
@martinpitt  force pushed  without fetching my remote changes, thus removing your commit by mistake, so I could not cherry-pick it later from anywhere so I stole it, (it appears as if I wrote it now, sorry :()

**Comment by martinpitt** [comment]
@KKoukiou all good, I'm glad to hand it to you :grin:

**Pull request merged**